lib/management/push.js

```diff
@@ -86,13 +84,17 @@ function startWSManagementClient(pushEndpoint, instanceId, token) {
         }
     }
 
+    overlayMessageListener = newOverlayCallback;
+
     const socketsByChannelId = [];
     const headers = {
         'x-instance-authentication-token': token,
     };
     const agent = createWSAgent(pushEndpoint, process.env, logger);
 
-    const url = `${pushEndpoint}/${instanceId}/ws`;
+    const url = process.env.ORBIT_PUSH_ENDPOINT ||
```

Comment:
The env var that already exists and is used by kube deployment is called `PUSH_ENDPOINT`

Reply:
Done